DESCRIPTION

```diff
@@ -0,0 +1,15 @@
+Package: hedgehog
+Type: Package
+Title: Property Testing for R
+Version: 0.1
+Date: 2017-08-14
+Authors@R: person("Huw", "Campbell", email = "huw.campbell@gmail.com",
+                  role = c("aut", "cre"))
+Description: Provides a modern property-based
+  testing system for R, in the spirit of QuickCheck.
+License: MIT + file LICENSE
+Imports: purrr
```

Comment:
I'm new to R and [purrr](http://purrr.tidyverse.org/) looks interesting 😃

Reply:
It is pretty cool yeah. Unfortunately though the `partial` function appears to be quite inefficient. I have timed state machine generation with and without it and it appears to be contributing about half the run time.